core/src/test/java/org/apache/iceberg/rest/requests/TestUpdateNamespacePropertiesRequest.java

```diff
@@ -206,7 +206,7 @@ public void testBuilderDoesNotCreateInvalidObjects() {
 
   @Override
   public String[] allFieldsFromSpec() {
-    return new String[] { "updates", "removals" };
+    return new String[] { "removals", "updates" };
```

Comment:
Updated to be in the order defined by the spec... can revert this change if we don't find it necessary.

Reply:
I think it would be best to revert this.

Reply:
Reverted